samples/server.js

```diff
@@ -33,8 +33,9 @@ app.get('/', (req, res) => {
 
 app.post('/log_payload', (req, res) => {
   // Log the request payload
-  console.log('Received task with payload: %s', req.body);
-  res.send(`Printed task payload: ${req.body}`).end();
+  const body = String(req.body);
```

Comment:
You shouldn't need to do the explicit cast here

Reply:
Hmmm, you're right. I must have incorrectly attributed a different problem to this. I'm going to revert all of my changes in this file.